x-pack/plugins/uptime/server/legacy_uptime/uptime_server.ts

```diff
@@ -62,6 +68,76 @@ export const initUptimeServer = (
     }
   });
 
+  legacyUptimePublicRestApiRoutes.forEach((route) => {
+    const { method, options, handler, validate, path } = uptimeRouteWrapper(
+      createRouteWithAuth(libs, route),
+      server
+    );
+
+    const routeDefinition = {
+      path,
+      validate,
+      options,
+    };
+
+    switch (method) {
+      case 'GET':
+        router.versioned
+          .get({
+            access: 'public',
+            path: routeDefinition.path,
+            options: {
+              tags: options?.tags,
+            },
+          })
+          .addVersion(
+            {
+              version: '2023-10-31',
```

Comment:
Defining a constant for a version and use it everywhere would be better as in:
https://github.com/elastic/kibana/blob/71bbbacd594885553ae10fc964ad1be9e0f63cf2/src/plugins/data_views/server/rest_api_routes/public/create_data_view.ts#L62